src/main/java/au/org/emii/ncdfgenerator/NcdfEncoder.java

```diff
@@ -228,10 +224,9 @@ else if( value instanceof Array )
 		}
 		// should we log here, or allow exceptions to propagate up?
 		catch ( Exception e ) {
-			// TODO log
 			StringWriter sw = new StringWriter();
 			e.printStackTrace(new PrintWriter(sw));
-			System.out.println( "Opps " + sw.toString() );
+			logger.error( "Problem generating netcdf " + sw.toString() );
```

Comment:
Logger.error(String msg, Throwable t) would remove the need to use a stringwriter